Imagine/Filter/Loader/RotateFilterLoader.php

```diff
@@ -0,0 +1,33 @@
+<?php
+
+namespace Liip\ImagineBundle\Imagine\Filter\Loader;
+
+use Imagine\Image\ImageInterface;
+use Imagine\Image\ManipulatorInterface;
+
+/**
+ * Loader for Imagine's basic rotate method
+ *
+ * @author Bocharsky Victor <bocharsky.bw@gmail.com>
+ */
+class RotateFilterLoader implements LoaderInterface
+{
+    /**
+     * Loads and applies a filter on the given image.
+     *
+     * @param ImageInterface $image
+     * @param array $options
+     *
+     * @return ManipulatorInterface
+     */
+    public function load(ImageInterface $image, array $options = array())
+    {
+        $angle = isset($options['angle']) ? intval(round(floatval($options['angle']), 0)) : 0;
```

Comment:
I am thinking we should not do anything here. The angle comes from config? A developer set it? So it is ok if he wants 0.5 angle. Let's just pass it as is to rotate method...

wdyt?